extensions/amp-script/0.1/amp-script.js

```diff
@@ -83,26 +107,37 @@ export class AmpScript extends AMP.BaseElement {
       }
       return [workerScript, authorScript, authorUrl];
     }),
-    // Configure callbacks.
+    // Options.
     {
-      onCreateWorker: data => {
-        dev().info(TAG, 'Create worker:', data);
-      },
-      onHydration: () => {
-        dev().info(TAG, 'Hydrated!');
-        this.element.classList.add('i-amphtml-hydrated');
-      },
-      onSendMessage: data => {
-        dev().info(TAG, 'To worker:', data);
+      status: this.status_,
```

Comment:
Possible confusion with [Window.status](https://developer.mozilla.org/en-US/docs/Web/API/Window/status)? Maybe we should have an obviously unique namespace for longTask and other custom APIs e.g. `global.workerDOM.foo`.

Reply:
I'd even be happy with `global.longTask.execute()`.